doc-files/release-notes-19.md

```diff
@@ -0,0 +1,91 @@
+# Release Notes for JavaFX 19
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+As of JDK 11 the JavaFX modules are delivered separately from the JDK. These release notes cover the standalone JavaFX 19 release. JavaFX 19 requires JDK 11 or later.
+
+## List of Enhancements
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8274771](https://bugs.openjdk.java.net/browse/JDK-8274771)|Map, FlatMap and OrElse fluent bindings for ObservableValue|base
+[JDK-8283346](https://bugs.openjdk.java.net/browse/JDK-8283346)|Optimize observable ArrayList creation in FXCollections|base
+[JDK-8286552](https://bugs.openjdk.java.net/browse/JDK-8286552)|TextFormatter: UpdateValue/UpdateText is called, when no ValueConverter is set|controls
+[JDK-8268225](https://bugs.openjdk.java.net/browse/JDK-8268225)|Support :focus-visible and :focus-within CSS pseudoclasses|graphics
+[JDK-8277309](https://bugs.openjdk.java.net/browse/JDK-8277309)|Add support for H.265/HEVC to HTTP Live Streaming|media
+
+## List of Fixed Bugs
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8290331](https://bugs.openjdk.java.net/browse/JDK-8290331)|Binding value left null when immediately revalidated in invalidation listener|base
```

Comment:
Should this be included? It was basically introduced in 19 and fixed in 19 (introduced by [JDK-8274771](https://bugs.openjdk.java.net/browse/JDK-8274771)|Map, FlatMap and OrElse fluent bindings for ObservableValue|base)

Reply:
That's a fair question. We generally have included these sort of bug fixes, although I realize that they are only relevant for those developers who picked up an early access build that had the bug.

Reply:
I prefer to keep this. I understand that this fixes a situation that could not occur on 18 or earlier, but still this is something that got changed in the course of 19, so to me it counts as an issue that is fixed and that would be noticeable if it wasn't fixed. 
If people or bots later parse the release notes, it could be confusing if they don't find this issue (it would be ok for issues with noreg-cleanup.)